packages/node/src/dev-server.ts

```diff
@@ -224,7 +224,7 @@ async function compileUserCode(
   } catch (error) {
     // We can't easily show a meaningful stack trace from ncc -> edge-runtime.
     // So, stick with just the message for now.
-    console.error(`Failed to instantiate edge runtime.`);
+    console.error(`Failed to compile user code for edge runtime.`);
```

Comment:
I changed this slightly to make it obvious when this is the error vs. the other location of exactly the same log line where we actually instantiate the edge runtime.